specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -359,9 +413,6 @@
             }
           }
         },
-        "x-ms-pageable": {
```

Comment:
What was wrong with this? Was it not pageable? If the value wasn't returned in a response, leave this here to avoid the breaking change and the client would never try to fetch the next page anyway (no way to know the link). Same for all instances below.

Reply:
This extension will have a conflict with Go SDK generation. It require Responses to be a collection if x-ms-pageable is set. However, the new responses blocks break this rule. I didn't find a good way to resolve this conflict. There is one similar issue before: https://github.com/Azure/autorest/issues/3494. The engineer removed this extension to solve the problem.

I went through this again and found I removed additional x-ms-pageable. There is only one cause this conflict but I removed four. I will add those back in this pr first and remove some of my code changes causing this conflict. We can discuss how to due with it if it is un-resolvable later.

p.s
https://github.com/Azure/azure-rest-api-specs/pull/11584
This closed pr is the previous pr I sent with this conflict.

Reply:
I added them back in the new commit.